apps/src/templates/VerticalImageResourceCard.jsx

```diff
@@ -71,17 +71,31 @@ class VerticalImageResourceCard extends Component {
     description: PropTypes.string.isRequired,
     buttonText: PropTypes.string.isRequired,
     link: PropTypes.string.isRequired,
+    image: PropTypes.string.isRequired,
     isRtl: PropTypes.bool.isRequired,
     MCShareLink: PropTypes.string
   };
 
   render() {
-    const { title, description, link, buttonText, isRtl, MCShareLink } = this.props;
+    const { title, description, link, buttonText, isRtl, MCShareLink, image } = this.props;
     const localeStyle = isRtl ? styles.rtl : styles.ltr;
 
+    const filenameToImgUrl = {
+      "another-hoc": require('@cdo/static/resource_cards/anotherhoc.png'),
```

Comment:
there's possibly room here to factor out the common path with a quick post-process "for each" on each of these.